modules/ROOT/pages/getting-started.adoc

```diff
@@ -77,21 +113,21 @@ build is "promoted" to update the references on the Component CR.
 === EnterpriseContractPolicy
 Building in {ProductName} follows a "build once, release multiple times" mentality where each release can have
 separate requirements on the builds before allowing the action. These build requirements are codified in an
-EnterpriseContractPolicy (ECP) which _may_ also be a CR. 
+EnterpriseContractPolicy (ECP).
 
 When an ECP is evaluated against a Snapshot, a single result is returned according to the highest violation. If,
-for example, all Components pass the policy requirements, the contract evaluation will be true. If a single
-Component in a Snapshot fails the policy, however, the result will be a failure even if all of the rest have
-clean passes. This behavior can present issues when running the default enterprise contract ITS when an Application
-contains multiple Components.
+for example, all Components pass the policy requirements, the policy evaluation is true. If a single
+Component in a Snapshot fails the policy, however, the overall result is a failure even if all of the rest have
+clean passes.
 
-NOTE: It is possible to enable Single Component mode when the ECP is being evaluated. See
-xref:patterns:testing-releasing-single-component.adoc[Testing and Release a Single Component]
+NOTE: This behavior can present issues when running the default enterprise contract ITS when an Application
+contains multiple Components. It is possible to enable "Single Component mode" when integration tests or release
+pipelines evaluate the ECP. See xref:patterns:testing-releasing-single-component.adoc[Testing and Release a Single Component]
```

Comment:
I know that we added this here to address a specific challenge. It doesn't quite feel like the right place to be though. Can we instead move this into Conforma sections?